java/code/src/com/redhat/rhn/domain/BaseDomainHelper.java

```diff
@@ -31,7 +32,7 @@
  * DB table: web_contact
  */
 @MappedSuperclass
-public abstract class BaseDomainHelper {
+public abstract class BaseDomainHelper implements Serializable {
```

Comment:
Did you check the sonarcloud reports about Serializable? This can grow big since all its fields need to be serializable too.

Reply:
I fixed them, (just one with `pillar` is left, but it would require lots of changes...I'd like to monitor it and be sure it's really required)